#ifndef RES_PJSIP_CALL_INCOMING_PUBLISH_CDS_START_H
#define RES_PJSIP_CALL_INCOMING_PUBLISH_CDS_START_H

#ifdef __cplusplus
extern "C" {
#endif // __cplusplus

void publish_cds_call_start(struct cobra_sip_ds_data *cobra_sip_data, struct nena_i3_ds_data *nena_i3_data);


#ifdef __cplusplus
}
#endif // __cplusplus

#endif // RES_PJSIP_CALL_INCOMING_PUBLISH_CDS_START_H
